### ISSUE TITLE
Add ability to pass in offset time when creating alerts via API

### ISSUE THREAD

**Issue by cklenk** [influxdata/influxdb-client-csharp#580]
__Proposal:__
Add offset to CreateThresholdCheckAsync and CreateDeadmanCheckAsync for alerting

__Current behavior:__
Unable to set offset time when using CreateThresholdCheckAsync and CreateDeadmanCheckAsync methods

__Desired behavior:__
Pass in optional offset time with functions

__Alternatives considered:__
Manually create alerts

__Use case:__
Ease of development and cleaner solution

**Comment by powersj** [comment]
Hi,

When you say offset time, what is your use-case for this offset? Is it a delay in notification?

Thanks

**Comment by cklenk** [comment]
Yes, that is correct. To account for late data that might come in.

Thanks,
Cameron

**Comment by powersj** [comment]
Based on the [API docs](https://docs.influxdata.com/influxdb/v2/api/#operation/GetChecks), it does look like the threshold type check has a `offset` parameter that is not exposed in this client library.

We can investigate this further, but would also be happy to see a PR with a fix to include this parameter.

Thanks!
